data/json/items/armor/ballistic_armor.json

```diff
@@ -408,27 +408,27 @@
     "copy-from": "ballistic_vest_esapi",
     "//": "based on smaller thorax only plate carriers",
     "name": { "str": "light ballistic vest (mag pouches)", "str_pl": "light ballistic vests (mag pouches)" },
-    "description": "A lightweight, simple plate carrier just covering the thorax with front and back pockets for armor plates.  This version has a MOLLE cummerbund that you can tuck a few mags in.",
+    "description": "A simple plate carrier covering the thorax, with front and back pockets for armor plates.  Designed to keep your organs from being perforated by small-arms fire.  This version has a MOLLE cummerbund that you can tuck a few magazines into.",
     "variant_type": "generic",
     "variants": [
       {
         "id": "ballistic_vest_light_olive",
         "name": { "str": "olive light ballistic vest (mag pouches)", "str_pl": "olive light ballistic vests (mag pouches)" },
-        "description": "It's a traditional army olive color.",
+        "description": "It's styled in olive camouflage.",
```

Comment:
A lot of small plate carriers are just a single color. I don't know if that still counts as camo.